dis_snek/models/discord_objects/user.py

```diff
@@ -423,6 +431,28 @@ def has_role(self, *roles: Union[Snowflake_Type, Role]) -> bool:
                 return False
         return True
 
+    async def timeout(
+        self, communication_disabled_until: Union["Timestamp", datetime, int, float, str, None], reason: str = MISSING
+    ):
+        """
+        Disable a members communication for a given time.
+
+        Args:
+            communication_disabled_until: The time until the user can communicate again
+            reson: The reason for this timeout
```

Comment:
```suggestion
            reason: The reason for this timeout
```
fix another parameter typo